contracts/JumpRateModelV2.sol

```diff
@@ -24,20 +25,9 @@ contract JumpRateModelV2 is BaseJumpRateModelV2 {
 
     }
 
-    /**
-     * @notice Calculates the current borrow rate per block
-     * @param cash The amount of cash in the market
-     * @param borrows The amount of borrows in the market
-     * @param reserves The amount of reserves in the market
-     * @param badDebt The amount of badDebt in the market
-     * @return The borrow rate percentage per block as a mantissa (scaled by 1e18)
-     */
-    function getBorrowRate(
-        uint256 cash,
-        uint256 borrows,
-        uint256 reserves,
-        uint256 badDebt
-    ) external view override returns (uint256) {
-        return _getBorrowRate(cash, borrows, reserves, badDebt);
+    /// @notice Get the number of slots/blocks in a year
+    /// @return Number of blocks in a year
+    function _getSlotsInOneYear() internal pure override returns (uint256) {
+        return BLOCKS_PER_YEAR;
```

Comment:
According to Commnet, it returns blocks annually. For other than bsc chains, don't we also require seconds annually?

Reply:
Since this is a virtual function specified in **BaseJumpRateModel**, it will yield the count of blocks in **JumpRateModelV2** and seconds in **TimeBasedJumpRateModelV2**.
I have fixed the netspec comments for it https://github.com/VenusProtocol/isolated-pools/pull/324/commits/aca35e56df275728c3ee83948cfe9164f049776e